MontyHallKata/Models/InputValidation.cs

```diff
@@ -0,0 +1,14 @@
+namespace MontyHallKata.Models
+{
+    public static class InputValidation
+    {
+        public static bool IsValidDoorSelection(int doorSelection)
+        {
+            return doorSelection is >= 0 and <= 3;
+        }
+        public static bool IsValidChoice(int choice)
+        {
+            return choice is >= 0 and <= 2;
+        }
+    }
+}
```

Comment:
It would be good to have some tests making sure the validators working properly, while they are quite simple.